var/spack/repos/spack_repo/builtin/packages/petsc/package.py

```diff
@@ -269,6 +269,10 @@ class Petsc(Package, CudaPackage, ROCmPackage):
     variant("hwloc", default=False, description="Activates support for hwloc")
     variant("kokkos", default=False, description="Activates support for kokkos and kokkos-kernels")
     variant("fortran", default=True, description="Activates fortran support")
+    # Install-time footprint is dominated by ~8 k tutorial/example files.
+    # Give packagers a switch to trim them away (‘spack install petsc ~examples’)
+    # while preserving current behaviour by default.
+    variant("examples", default=True, description="Install tutorial sources and libraries.")
```

Comment:
I'm not sure if there is a better convention to translate 'make install-lib' to spack notation. Here 'examples' name might be a bit over-loaded. Perhaps:

```suggestion
    variant("install-examples", default=True, description="Install test and tutorial example sources.")
```